main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -54,8 +62,21 @@ public static void main(String[] argv) {
         NoticeContainer noticeContainer = new NoticeContainer();
         GtfsFeedContainer feedContainer;
         try {
-            feedContainer = feedLoader.loadAndValidate(GtfsInput.createFromPath(args.input), feedName, validatorLoader,
-                    noticeContainer);
+            if (args.input == null) {
+                feedContainer = feedLoader.loadAndValidate(
+                        GtfsInput.createFromUrl(
+                                new URL(args.url),
+                                args.storageDirectory),
+                        feedName,
+                        validatorLoader,
+                        noticeContainer);
+            } else {
+                feedContainer = feedLoader.loadAndValidate(
+                        GtfsInput.createFromPath(args.input),
+                        feedName,
+                        validatorLoader,
+                        noticeContainer);
+            }
```

Comment:
The following is just a try at a multiline suggestion in the GitHub interface with @aababilov [comment ](https://github.com/MobilityData/gtfs-validator/pull/541/files/2cf8c83c24bad5cae84d73c204ccfb369ba14ea1#r537161917)'s content

```suggestion
            GtfsInput gtfsInput = args.input == null ? GtfsInput.createFromUrl(
                                new URL(args.url),
                                args.storageDirectory)
  : GtfsInput.createFromPath(args.input);

feedContainer = feedLoader.loadAndValidate(
                        gtfsInput,
                        feedName,
                        validatorLoader,
                        noticeContainer);
```